pkg/agent/datapath/multiBridgeDatapath.go

```diff
@@ -1095,8 +1095,34 @@ func (datapathManager *DpManager) removeIntenalIP(ip string, index int) {
 	}
 }
 
+func (datapathManager *DpManager) getFlush() bool {
+	datapathManager.flushMutex.Lock()
+	defer datapathManager.flushMutex.Unlock()
+	return datapathManager.needFlush
+}
+
+func (datapathManager *DpManager) setFlush(needFlush bool) {
+	datapathManager.flushMutex.Lock()
+	defer datapathManager.flushMutex.Unlock()
+	datapathManager.needFlush = needFlush
+}
+
 func (datapathManager *DpManager) cleanConntrackWorker() {
 	for {
+		datapathManager.flushMutex.Lock()
+		if datapathManager.needFlush {
+			err := netlink.ConntrackTableFlush(netlink.ConntrackTable)
+			if err != nil {
+				klog.Errorf("Flush ct failed: %v", err)
+			} else {
+				datapathManager.needFlush = false
+				klog.Info("Success flush ct")
+			}
+			datapathManager.flushMutex.Unlock()
+			continue
+		}
+		datapathManager.flushMutex.Unlock()
```

Comment:
we could put L1112 inside if, and optimize L1121-L1124 into single unlock. Replace needFlush with getFlush()  L1113